.github/workflows/trivy.yml

```diff
@@ -0,0 +1,33 @@
+name: Update Trivy Cache
```

Comment:
The same comment - let's rename it, since the check itself is performed in ci.yml workflow

Reply:
Moved it to `trivy-db-cache.yml`